client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -252,7 +253,20 @@ class CampaignAdminList extends Component {
     }
   }
 
+  renderViewModeToggle() {
+    const { ViewModeComponent } = this.props;
+
+    return (
+      <ViewModeComponent
+        id="view-mode-toggle-in-edit-nb"
+        area={'edit'}
+      />
+    );
+  }
```

Comment:
It looks like this method isn't used at all - perhaps you could remove it. The component is rendered in the `render()` method directly and doesn't need to be found to the instance in construction I don't think